app/services/activities/import_from_csv.rb

```diff
@@ -121,6 +121,7 @@ class ActivityCreator
         :collaboration_type_step,
         :flow_step,
         :aid_type_step,
+        :ftsc_step,
```

Comment:
Should this be `fstc_applies_step` instead?

Reply:
👍